requirements.txt

```diff
@@ -0,0 +1,130 @@
+anyio==4.3.0
+appnope==0.1.4
+argon2-cffi==23.1.0
+argon2-cffi-bindings==21.2.0
+arrow==1.3.0
+asttokens==2.4.1
+async-lru==2.0.4
+attrs==23.2.0
+Babel==2.14.0
+beautifulsoup4==4.12.3
+bleach==6.1.0
+Bottleneck==1.3.7
+catboost==1.2
+certifi==2024.2.2
+cffi==1.16.0
+charset-normalizer==3.3.2
+comm==0.2.1
+contourpy==1.2.0
+cycler==0.11.0
+debugpy==1.8.1
+decorator==5.1.1
+defusedxml==0.7.1
+exceptiongroup==1.2.0
+executing==2.0.1
+fastjsonschema==2.19.1
+fonttools==4.25.0
+fqdn==1.5.1
+graphviz==0.20.1
+h11==0.14.0
+httpcore==1.0.4
+httpx==0.27.0
+idna==3.6
+ipykernel==6.29.3
+ipython==8.22.1
+ipywidgets==8.1.2
+isoduration==20.11.0
+jedi==0.19.1
+jenkspy==0.4.0
+Jinja2==3.1.3
+joblib==1.3.2
+json5==0.9.17
+jsonpointer==2.4
+jsonschema==4.21.1
+jsonschema-specifications==2023.12.1
+jupyter==1.0.0
+jupyter_client==8.6.0
+jupyter-console==6.6.3
+jupyter_core==5.7.1
+jupyter-events==0.9.0
+jupyter-lsp==2.2.3
+jupyter_server==2.12.5
+jupyter_server_terminals==0.5.2
+jupyterlab==4.1.2
+jupyterlab_pygments==0.3.0
+jupyterlab_server==2.25.3
+jupyterlab_widgets==3.0.10
+kiwisolver==1.4.4
+MarkupSafe==2.1.5
+matplotlib==3.8.0
+matplotlib-inline==0.1.6
+mistune==3.0.2
+mkl-fft==1.3.1
+mkl-random==1.2.2
+mkl-service==2.4.0
```

Comment:
I was not able to install these packages and had to comment them out. The error is:

```ERROR: Could not find a version that satisfies the requirement mkl-fft (from versions: none)
ERROR: No matching distribution found for mkl-fft```

for `mkl-fft` and same issue for `mkl-random` and `mkl-service`